coderd/database/queries/workspaces.sql

```diff
@@ -70,7 +70,8 @@ FROM
 WHERE
 	owner_id = @owner_id
 	AND deleted = @deleted
-	AND LOWER("name") = LOWER(@name);
+	AND LOWER("name") = LOWER(@name)
```

Comment:
😅

Reply:
Wait I'm confused what this means :thinking:

Reply:
oh, sorry @Emyrk, this wasn't actionable and was actually directed at the sorting line below.